components/ui/Footer.js

```diff
@@ -1,53 +1,73 @@
-"use client"
+"use client";
 
-import { usePathname } from 'next/navigation';
-import styles from './Footer.module.css';
+import { usePathname } from "next/navigation";
+import styles from "./Footer.module.css";
+import Link from "next/link";
+import data from "@/data/galectica.json";
+import SocialmediaItem from "./socialMediaItem";
+import {
+  faLinkedin,
+  faFacebook,
+  faInstagram,
+  faTiktok,
+  faGoogle,
+} from "@fortawesome/free-brands-svg-icons";
+import { library } from "@fortawesome/fontawesome-svg-core";
 
 export const Footer = () => {
-  const path = usePathname().split('?')[0];
+  const path = usePathname().split("?")[0];
   return (
     <footer className={path !== "/" ? styles.footer : styles.hidden}>
       <div className={styles.footerDescription}>
         <h3>Galactica</h3>
-        <p>Explore the universe and beyond. Your journey to the stars starts here.</p>
+        <p>
+          Explore the universe and beyond. Your journey to the stars starts
+          here.
+        </p>
         <p>&copy; 2024 Galactica. All rights reserved.</p>
       </div>
-      {/* TASK - React 1 week 2 */}
-      {/* Create a new List for the Pages */}
-      {/* We need to use the <Link /> component here */}
-      {/* <div className={styles.pages}>
-        <h3>Pages</h3>
-        <ul>
-          <li> <Link/> </li>
-          ...
-        </ul>
-      </div> */}
-      {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
-
-      {/* TASK - React 1 week 1 */}
-      {/* Add a new list item for LINKEDIN */}
       <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
+        <h3>Pages</h3>
         <ul className={styles.footerList}>
           <li>
-            <a href="https://facebook.com">Facebook</a>
+            {" "}
```

Comment:
```suggestion
            
```
You don't need this extra empty space here

Reply:
done